Dockerfile

```diff
@@ -11,9 +11,13 @@
 # instead of the default. See docs-developer/docker.md for more information.
 
 # Setup a container and build the project in it
-# We set up a node 10 running in latest Debian stable called buster, in the
+# We set up a node 12 running in latest Debian stable called buster, in the
 # "slim" flavor because we don't need the big version.
-FROM node:12-buster-slim AS builder
+# By specifying an exact node version, we let dependabot update the version when
+# there's a new one, so that a new docker image is generated. Make sure to use
+# the exact same version for the second part of the Dockerfile.
+# /!\ Make sure to update dependabot.yml when switching to a new major version. /!\
+FROM node:12.18.0-buster-slim AS builder
```

Comment:
I specify a complete version number now. Thanks to dependabot this will force us to update the docker image when there's a new node LTS version. Note there are more update happening, like every day, because in the buster image itself dependencies are updated. So this is a good thing that we get regular updates.

I used 12.18.0 because there _is_ a 12.18.1 version and I want to see if dependabot will update it correctly, in both places in this file.